pvlib/floating.py

```diff
@@ -0,0 +1,66 @@
+"""
+The ``floating`` module contains functions for calculating parameters related
+to floating PV systems.
+"""
+
+import numpy as np
+import pandas as pd
+
+
+def stream_temperature_stefan(temp_air):
+    r"""
+    Estimate daily stream water temperature from daily ambient air temperature.
+
+    Parameters
+    ----------
+    temp_air : numeric
+        Daily average ambient dry bulb temperature. [°C]
+
+    Returns
+    -------
+    water_temperature : numeric
+        Daily average stream water temperature. [°C]
+
+    Notes
+    -----
+    Daily average stream water temperature
+    :math:`T_w` is calculated from daily ambient air temperature
+    :math:`T_{air}` as provided in [1]_:
+
+    .. math::
+       :label: stream
```

Comment:
```suggestion
```

Unnecessary unless you want to reference it, or need automatic equation numbering - does not seem to be the case.